publisher/product/ontology/build-about.sh

```diff
@@ -86,10 +78,10 @@ __HERE__
     "${JENA_ARQ}" \
       --data="${tmpAboutFileDev}" \
       --query="${SCRIPT_DIR}/lib/echo.sparql" \
-      --results=RDF > "${tag_root}/LoadFIBODev.rdf" 2> "${TMPDIR}/err.tmp"
+      --results=RDF > "${tag_root}/Load${ONTPUB_FAMILY}Dev.rdf" 2> "${TMPDIR}/err.tmp"
 
     if [ -s "${TMPDIR}/err.tmp" ] ; then
-      warning "no RDF XML output generated.  Use LoadFIBODev.ttl file instead"
+      warning "no RDF XML output generated.  Use Load${ONTPUB_FAMILY}Dev.ttl file instead"
```

Comment:
Should be uppercase:
`warning "no RDF XML output generated. Use Load${ONTPUB_FAMILY^^}Dev.ttl file instead"`